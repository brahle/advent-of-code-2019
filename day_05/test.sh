#!/bin/bash
./machine.sh < test/test.$1.in | grep -e "\(OUTPUT\|DONE\)" > out

diff -q test/test.$1.out out
if [ $? -ne 0 ]
then
    echo "EXPECTED: "
    cat test/test.$1.out

    echo ""
    echo "GOT: "
    cat out
    exit 1
fi
