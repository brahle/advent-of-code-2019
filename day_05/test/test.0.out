DONE 3654868
